care/facility/models/patient_consultation.py

```diff
@@ -129,9 +129,8 @@ class PatientConsultation(PatientBaseModel, ConsultationRelatedPermissionMixin):
     admission_date = models.DateTimeField(null=True, blank=True)  # Deprecated
     icu_admission_date = models.DateTimeField(null=True, blank=True)
     discharge_date = models.DateTimeField(null=True, blank=True)
-    discharge_reason = models.CharField(
+    discharge_reason = models.SmallIntegerField(
         choices=DISCHARGE_REASON_CHOICES,
```

Comment:
```suggestion
        choices=DiseaseReasonEnum.choices,
```

PS: in django 5.0+ accessing the `.choices` won't be required [ref](https://docs.djangoproject.com/en/5.0/releases/5.0/#more-options-for-declaring-field-choices)